applications/plugins/SofaCUDA/CMakeLists.txt

```diff
@@ -334,6 +334,7 @@ sofa_find_package(Sofa.GUI.Qt QUIET)
 if(Sofa.GUI.Qt_FOUND)
     list(APPEND HEADER_FILES sofa/gpu/gui/CudaDataWidget.h)
     list(APPEND SOURCE_FILES sofa/gpu/gui/CudaDataWidget.cpp)
+    list(APPEND SOURCE_FILES sofa/gpu/gui/CudaMouseInteraction.cpp)
```

Comment:
I don't see how it is related to Qt

Reply:
I'm sorry, I was trying to compile without the QT gui and didn't manage to do it because of this component. But it was more due to a side effect rather than a direct dependency on Qt. In fact this component depends on `Sofa.GUI` that was linked through `Sofa.GUI.Qt`. So if it wasn't found, the `Sofa.GUI` library was never linked. I've committed the changes.